qa/suites/fs/upgrade/featureful_client/old_client/overrides/multimds/no.yaml

```diff
@@ -1,3 +1,4 @@
 overrides:
   ceph:
-    max_mds: 1
+    cephfs:
+      max_mds: 1
```

Comment:
We're also missing this change in several other yaml fragments like `qa/cephfs/clusters/9-mds.yaml`.

Reply:
Missed them. Fixed already. Thanks.